homeassistant/components/mqtt/manifest.json

```diff
@@ -4,7 +4,7 @@
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/mqtt",
   "requirements": ["paho-mqtt==1.6.1"],
-  "dependencies": ["http"],
+  "dependencies": ["http", "repairs"],
```

Comment:
Should we add a repairs issue too? Otherwise there's no need for this.

Reply:
I agree,  #77174 will be the follow up. Closing this PR.